topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -0,0 +1,299 @@
+>    - "Plot per chromosome" to `False`
+>    - "Chromosomes to include (and order to plot in)" to `chr1 chr2 chr3 chr4 chr5 chr6 chr7 chr8 chr9 chr10 chr11 chr12 chr13 chr14 chr15 chr16 chr17 chr18 chr19 chrX chrY`
+>    - "Plot only this region" to `chrX` 'or' `X`
+>    - "Plot the log1p of the matrix values" to `True`
+>
+{: .hands_on}
+
+# TAD calling
+
+“The partitioning of chromosomes into topologically associating domains (TADs) is an emerging concept that is reshaping our understanding of gene regulation in the context of physical organization of the genome” [Ramirez et al. 2017](https://doi.org/10.1101/115063).
+
+TAD calling works in two steps: First HiCExplorer computes a TAD-separation score based on a z-score matrix for all bins. Then those bins having a local minimum of the TAD-separation score are evaluated with respect to the surrounding bins to decide assign a p-value. Then a cutoff is applied to select the bins more likely to be TAD boundaries.
+
+[hicFindTADs](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicFindTADs.html#hicfindtads) tries to identify sensible parameters but those can be change to identify more stringent set of boundaries.
+
+> ### {% icon hands_on %} Hands-on: Finding TADs
+>
+> 1. **hicFindTADs** {% icon tool %}: Run hicFindTADs on the corrected matrix adjusting the parameters:
+>    - "Minimum window length (in bp) to be considered to the left and to the right of each Hi-C bin." to `60000`
+>    - "Maximum window length (in bp) to be considered to the left and to the right of each Hi-C bin." to `120000`
+>    - "Step size when moving from minDepth to maxDepth" to `20000`
+>    - "Multiple Testing Corrections" to `False discovery rate`
+>    - "q-value" to `0.05` 
+>    - "Minimum distance between boundaries (in bp)." to `80000`
+>
+{: .hands_on}
+
+As an output we get the boundaries, domains and scores separated files. We will use in the plot later only the TAD-score file.
+
+We can plot the TADs for a given chromosomal region. For this we need to create a track file containing the instructions to build the plot. The [hicPlotTADs](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicPlotTADs.html) documentation contains the instructions to build the track file.
+
+In following plot we will use the listed track file. Please store it as track.ini with Upload Manager.
+
+```[hic]
+file = hicMatrix/replicateMerged.Corrected_20kb.h5
+title = HiC mESC chrX:99974316-101359967
+colormap = RdYlBu_r
+depth = 2000000
+width = 7
+transform = log1p
+x labels = yes
+type = interaction
+file_type = hic_matrix
+boundaries_file =  TADs/marks_et-al_TADs_20kb-Bins_domains.bed
+
+[x-axis]
+fontsize=16
+where=top
+
+[tad score]
+file = TADs/marks_et-al_TADs_20kb-Bins_score.bedgraph
+title = "TAD separation score"
+width = 2
+type = lines
+color = blue
+file_type = bedgraph
+
+[spacer]
+
+[gene track]
+file = mm10_genes_sorted.bed
+width = 10
+title = "mm10 genes"
+width = 5
+labels = off
+```
+

```

Comment:
We can not use a `track.ini` file on Galaxy, this would be too complicate for the most users. Please have a look at the wrapper. For each track you can select what you want to plot. In this case use `TAD visualization` with our matrix as `matrix to compute on`, and the mentioned boundaries file from hicFindTADs as `Boundaries file`. To plot the tad score, use `TAD score` as `Choose style of the track` and for the gene track it is `Gene track`. 

Mention that the gene tracks needs to be imported from the data library.

Reply:
The range is X:98000000-105000000

Reply:
Range is now: chr2L:1000000-5000000